log4j-mongodb/src/test/java/org/apache/logging/log4j/mongodb/MongoDbAuthFailureIT.java

```diff
@@ -29,16 +29,18 @@
 import org.junit.jupiter.api.Test;
 
 @UsingMongoDb
-@LoggerContextSource("log4j2-mongodb-auth-failure.xml")
-public class MongoDbAuthFailureTest {
+@LoggerContextSource("MongoDbAuthFailureIT.xml")
+class MongoDbAuthFailureIT {
```

Comment:
Can we suppress the output of this test, please? Tried adding `@UsingStatusListener`, but did not help.

Reply:
I silenced the tests in https://github.com/apache/logging-log4j2/pull/2922/commits/bbd4e7df620331c297049035cd70488ce17d5aeb.

`@UsingStatusListener` does not silence the status logger, it keeps its level the same as configured. In case of failure, however, it prints all the messages at least as severe `DEBUG`

Fixed in https://github.com/apache/logging-log4j2/pull/2922/commits/bbd4e7df620331c297049035cd70488ce17d5aeb